app/src/main/java/io/intrepid/contest/screens/entrysubmission/entryimage/EntryImageActivity.java

```diff
@@ -65,6 +65,8 @@ protected void onViewCreated(Bundle savedInstanceState) {
         if (!getPackageManager().hasSystemFeature(PackageManager.FEATURE_CAMERA)) {
             chooseCameraButton.setEnabled(false);
         }
+
+        setActionBarTitle(getResources().getString(R.string.entry_image_bar_title));
```

Comment:
I am a bit confused... overload `setActionBar` in this class, or overload `setActionBarTitle` (which I added myself to the [BaseActivity class](https://github.com/IntrepidPursuits/ContestAndroid/blob/gabi/more-look-and-feel/app/src/main/java/io/intrepid/contest/base/BaseActivity.java#L89))?

And just to confirm, overload because I should keep using this when my string resource is [built dynamically](https://github.com/IntrepidPursuits/ContestAndroid/pull/13/files#diff-9420d6573f9d9b3821adfe82b919922bR44)?